layouts/_default/scaler-template.html

```diff
@@ -0,0 +1,26 @@
+<div id="" class="column is-one-third">
+  <div class="card">
+    <div class="card-content">
+      <a
+        class="title is-size-4 is-size-5-mobile has-text-primary has-text-weight-bold scaler-title"
+      >
+      </a>
+
+      <br /><br />
+
+      <div class="content is-medium description"></div>
+    </div>
+
+    <footer class="card-footer">
+      <div class="card-footer-item is-spaced card-is-wrapped">
+        Availability&nbsp;<span
+          class="has-text-weight-bold availability"
+        ></span>
+      </div>
+
+      <div class="card-footer-item card-is-wrapped">
+        Maintainer:&nbsp;<span class="has-text-weight-bold maintainer"></span>
+      </div>
+    </footer>
+  </div>
+</div>
```

Comment:
This shouldn't be under `_default`. Did you try putting it under `partials`?

Reply:
oh wow...just saw this now. 
Would try making it a partial instead. 
Thanks for bringing this to my attention!

Reply:
No worries. Great job overall! Please cc me on your refactoring-into-partial PR. Thanks @thisisobate!